assignments/01-ownership-borrowing/src/mutables.rs

```diff
@@ -1,16 +1,27 @@
+
 /// Increments a mutable i32 reference by 1. On overflow, wraps around.
-fn increments() {
-    todo!();
+fn increments(input: &mut i32) {
+    if *input == i32::MAX {
+        *input = i32::MIN;
+    } else {
+        *input += 1;
+    }
 }
 
 /// Given two mutable i32 references, swap their values
-fn swap() {
-    todo!();
+fn swap(one: &mut i32, two: &mut i32) {
+    let temp = *one;
+    std::mem::swap(&mut (*one), &mut (*two));
```

Comment:
this is the only line you need - the others are unnecessary (& are actually messing with your swap)